build.gradle

```diff
@@ -44,4 +46,8 @@ subprojects { subProject ->
             languageVersion = "1.5"
         }
     }
+
+    tasks.withType(Test).configureEach {
+        timeout.set(Duration.ofMinutes(2))
+    }
```

Comment:
I added this timeout to prevent the tests from hanging. I think 2 minutes should be fine.

Reply:
Nice, thanks!